example/ios/example/Info.plist

```diff
@@ -24,6 +24,25 @@
 	<string>1</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+	<key>NSAppTransportSecurity</key>
+	<dict>
+		<key>NSExceptionDomains</key>
+		<dict>
+			<key>localhost</key>
+			<dict>
+				<key>NSExceptionAllowsInsecureHTTPLoads</key>
+				<true/>
+			</dict>
+		</dict>
+	</dict>
+	<key>NSCameraUsageDescription</key>
+	<string>OpenTok wants to use your camera, do you want to allow it?</string>
+	<key>NSLocationWhenInUseUsageDescription</key>
```

Comment:
Is it needed?

Reply:
Yeah, otherwise the app on real device crash.

Reply:
I think @dratwas was speaking about `NSLocationWhenInUseUsageDescription`, which I don't think is needed.